SAMtags.tex

```diff
@@ -250,29 +250,23 @@ \subsection{Barcodes}
 The {\tt BC} tag should match the {\tt QT} tag in length. 
 In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and places a hyphen (`{\tt -}') between the barcodes from the same template. 
 
-\item[BZ:Z:\tagvalue{qualities+}] 
-Phred quality of the (uncorrected) unique molecular identifier sequence in the {\tt OX} tag.
+\item[QT:Z:\tagvalue{qualities}] 
+Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag. 
 Same encoding as {\sf QUAL}, i.e., Phred score + 33.
-The {\tt OX} tags should match the {\tt BZ} tag in length. 
-In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with a space (`{\tt \textvisiblespace}') between the different strings.
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with spaces (`{\tt \textvisiblespace}') between the different strings from the same template. 
```

Comment:
should be sample-barcodes, no?

Reply:
Yes, I think so. The issue was already in the tex file before this PR. Shall I include the fix in this PR?